components/popover/popover-mixin.js

```diff
@@ -278,7 +292,7 @@ export const PopoverMixin = superclass => class extends superclass {
 	}
 
 	configure(properties) {
-		this._margin = properties?.margin ?? 18;
+		this._margin = properties?.margin ?? ((properties?.position?.location === 'inline-start' || properties?.position?.location === 'inline-end') ? 0 : 18);
```

Comment:
nit: maybe some sort of enum could be useful for the position location options since there are a lot of hard coded strings

Reply:
Thanks. Yeah the use of the strings gradually grew. Thanks for the callout.